source/update.rst

```diff
@@ -6,17 +6,26 @@ Update
    As for every update process, you have to backup some data before processing any upgrade:
 
    * **backup your database**;
-   * backup your files directory;
-   * backup your configuration (especially your GLPI key file `config/glpi.key` or `config/glpicrypt.key` if present).
+   * backup your `files` directory;
+   * backup your `config` directory, especially for your GLPI key file (`config/glpi.key` or `config/glpicrypt.key`) which is randomly generated.
```

Comment:
Maybe add a line about plugins and/or marketplace directories as well?

Reply:
done